docs/source/terms_and_model.rst

```diff
@@ -1147,3 +684,965 @@ The GA4GH computed identifier for these sets is
+
+.. _Gene:
+
+Gene
+####
+
+A gene is a basic and fundamental unit of heritability. Genes are
+functional regions of heritable DNA or RNA that include transcript
+coding regions, regulatory elements, and other functional sequence
+domains. Because of the complex nature of these many components
+comprising a gene, the interpretation of a gene is context dependent.
+
+**Computational definition**
+
+A gene is an authoritative representation of one or more heritable
+:ref:`Locations <Location>` that includes all sequence elements
+necessary to perform a biological function. A gene may include
+regulatory, transcribed, and/or other functional Locations.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - gene_id
+     - :ref:`CURIE`
+     - 0..1
+     - Authoritative Gene ID (see guidance)
+   * - type
+     - string
+     - 1..1
+     - MUST be set to **`Gene`**
+
+**Implementation guidance**
+
+* Gene symbols (e.g., "BRCA1") are unreliable keys.  Implementations
+  MUST NOT use a gene symbol to define a Gene.
+* A gene is specific to a species.  Gene orthologs have distinct
+  records in the recommended databases.  For example, the BRCA1 gene
+  in humans and the Brca1 gene in mouse are orthologs and have
+  distinct records in the previously recommended gene databases.
+* Implementations MUST use authoritative gene namespaces available from
+  identifiers.org whenever possible.  Examples include:
+
+    * `hgnc <https://registry.identifiers.org/registry/hgnc>`__
+    * `ncbigene <https://registry.identifiers.org/registry/ncbigene>`__
+    * `ensembl <https://registry.identifiers.org/registry/ensembl>`__
+    * `vgnc <https://registry.identifiers.org/registry/vgnc>`__
+    * `mgi <https://registry.identifiers.org/registry/mgi>`__
+* The `hgnc` namespace is RECOMMENDED for human
+  variation in order to improve interoperability.
+* Gene MAY be converted to one or more :ref:`Locations <Location>`
+  using external data. The source of such data and mechanism for
+  implementation is not defined by this specification.
+
+**Example**
+
+The following examples all refer to the human BRCA1 gene:
+
+.. parsed-literal::
+
+   {
+     'gene': 'ncbigene:672',
+     'type': 'Gene'
+   }
+
+   {
+     'gene': 'hgnc:1100',
+     'type': 'Gene'
+   }
+
+   {
+     'gene': 'ensembl:ENSG00000012048',
+     'type': 'Gene'
+   }
+
+
+**Sources**
+
+* `SequenceOntology: gene (SO:0000704)
+  <http://www.sequenceontology.org/browser/current_release/term/SO:0000704>`__
+  — A region (or regions) that includes all of the sequence elements
+  necessary to encode a functional transcript. A gene may include
+  regulatory regions, transcribed regions and/or other functional
+  sequence regions.
+
+.. _Quantity:
+
+Quantity
+@@@@@@@@
+
+A value indicating a multitude or magnitude measure.
+
+.. _CopyCount:
+
+CopyCount
```

Comment:
We should consider externalizing the `absolute_measure` boolean flag from this construct as it does not make sense for certain uses (e.g. inside RepeatedSequence).  It may make more sense to provide two conditional attributes on the CopyNumber concept (AbsoluteCopyCount or RelativeCopyCount) or have a single CopyCount attribute and a boolean `absolute_measure` that sits at the CopyNumber level to handle this specific need.